assets/aml-benchmark/tests/dataset_preprocessor/test_dataset_preprocessor.py

```diff
@@ -347,9 +347,6 @@ def test_invalid_inputs(
         invalid_dataset_error_mssg = (
             "the following arguments are required: --dataset"
         )
-        invalid_jsonl_dataset_mssg = (
```

Comment:
We would still need tests when no "valid" jsonl file is passed in?

Reply:
@arun-rajora Why would we need it? This test was for the input_validation method in dataset_preprocessor and inference postprocessor component.